geode-core/src/main/java/org/apache/geode/internal/cache/control/SerializableRestoreRedundancyResultsImpl.java

```diff
@@ -39,7 +39,8 @@
   public void addPrimaryReassignmentDetails(PartitionRebalanceInfo details) {
     totalPrimaryTransfersCompleted += details.getPrimaryTransfersCompleted();
     totalPrimaryTransferTime =
-        totalPrimaryTransferTime.plusMillis(details.getPrimaryTransferTime());
+        totalPrimaryTransferTime + details.getPrimaryTransferTime();
```

Comment:
This could be replaced with `totalPrimaryTransferTime += details.getPrimaryTransferTime();`

Reply:
done